pixi.toml

```
@@ -0,0 +1,94 @@
+[project]
+name = "imod-python"
+version = "0.15.1"
+description = "Make massive MODFLOW models!"
```

Minor nitpick: This is already a bold statement, the ``!`` is a bit too much ;-)

```suggestion
description = "Make massive MODFLOW models"
```

---

Done